pkg/etcdutil/layout.go

```diff
@@ -64,12 +63,12 @@ func TaskMasterPath(appName string, taskID uint64) string {
 	return path.Join("/", appName, TasksDir, strconv.FormatUint(taskID, 10), TaskMaster)
 }
 
-func MetaPath(linkType, appName string, taskID uint64) string {
+func MetaPath(appName string, taskID uint64) string {
 	return path.Join("/",
 		appName,
 		TasksDir,
 		strconv.FormatUint(taskID, 10),
-		linkType)
+		"meta")
```

Comment:
I think we can pass the meta in.

However, we need to define link type first. Is it some way for app to define a group of neighbors? 
Because that's what the interface says:
https://github.com/taskgraph/taskgraph/blob/master/topology_interface.go#L26
https://github.com/taskgraph/taskgraph/blob/master/framework_interface.go#L47